src/libs/actions/App.js

```diff
@@ -208,6 +207,35 @@ function reconnectApp(updateIDFrom = 0) {
     });
 }
 
+/**
+ * Fetches data when the app will call reconnectApp without params for the last time. This is a separate function
+ * because it will follow patterns that are not recommended so we can be sure we're not putting the app in a unusable
+ * state because of race conditions between reconnectApp and other pusher updates being applied at the same time.
+ * @return {Promise}
+ */
+function lastReconnectAppAfterActivatingReliableUpdates() {
```

Comment:
I don't feel great about this name, but I don't have much better of a suggestion.

Reply:
Oh, maybe `finalReconnectAppAfter...`

Reply:
Changed